### ISSUE TITLE
Merge in 7.x-5.0

### ISSUE THREAD

**Issue by jackaponte** [backdrop-contrib/webform_civicrm#43]
https://github.com/colemanw/webform_civicrm/releases/tag/7.x-5.0
https://github.com/colemanw/webform_civicrm/compare/7.x-4.28...7.x-5.0

**Comment by jackaponte** [comment]
I never got to [that bug fix release I'd hoped for,](https://github.com/backdrop-contrib/webform_civicrm/issues/40#issuecomment-555184241) but I plan to tackle this merge and a new release next week!

**Comment by herbdool** [comment]
Cool, thanks @jackaponte! Have you tried out the latest release? It seems like the 5.x indicates it works with latest Civi versions only and that it's focused on working to Webform in Drupal 8. But that it still works with Webform in Backdrop.

**Comment by jackaponte** [comment]
> Have you tried out the latest release? It seems like the 5.x indicates it works with latest Civi versions only and that it's focused on working to Webform in Drupal 8. But that it still works with Webform in Backdrop.

Ah, thanks for highlighting these notes, I hadn't looked so carefully yet and haven't tried the latest release. From what I read on the [Drupal.org project page](https://www.drupal.org/project/webform_civicrm) it should work okay for both Drupal 7 and Backdrop. We'll need to update [the README](https://github.com/backdrop-contrib/webform_civicrm#version-compatibility) to reflect compatibility changes though!

**Comment by jackaponte** [comment]
I've created a new PR, #44. Please review it when you get a chance, @herbdool!

I've not yet updated the version compatibility section of the README. It seems to me that we need either a full-on version compatibility table like the one on the D.o project page, or we need to point people to that project page and explain how to translate Drupal versions to Backdrop versions. Any thoughts on which would work better and/or be easier to keep updated?

**Comment by herbdool** [comment]
I'll take a look at the PR soon. Thanks @jackaponte!

I think we can keep the compatibility pretty simple. We can change the line to:

> This module's 4.x version is compatible with CiviCRM 4.4 through 5.x. The 5.x version is compatible with CiviCRM 5.12+.

We don't even need to mention Backdrop or Webform since they both only have one major version at this point.

**Comment by herbdool** [comment]
The PR looks good @jackaponte. I haven't tested but sounds like you have. I think it's good to go, other than updating the line about compatibility. After looking in the code it mentions that the minimum Webform version is 1.x-4.19.0. Does it mention 4.19 in the original too? If so then I guess we need to edit the line above to say that too.

**Comment by jackaponte** [comment]
> After looking in the code it mentions that the minimum Webform version is 1.x-4.19.0. Does it mention 4.19 in the original too? 

If by "the original" you mean the Drupal 7.x-5.0 release, then yes! That change to webform_civicrm.info was made back in April 2019 in [commit `e09cf8a32ba085d4e06eb2789acfee2e900ba6cc`](https://github.com/colemanw/webform_civicrm/commit/e09cf8a32ba085d4e06eb2789acfee2e900ba6cc) so it's been around in our Backdrop version for a while now, too.

> If so then I guess we need to edit the line above to say that too.

I went with this, to be as clear as possible at the risk of being redundant:

> The 1.x-4.x version is compatible with Backdrop 1.x, Webform 1.x-4.19+ and CiviCRM 4.4 through 5.x. The 1.x-5.x version is compatible with Backdrop 1.x, Webform 1.x-4.19+ and CiviCRM 5.12+.

**Comment by jackaponte** [comment]
Welp I made a bit of a mess of the merge, hence the reversions, but I've merged the Drupal 7.x-5.0 updates into our Backdrop 1.x-5.x branch, which I've made the default branch for the repo.

I also successfully [created a new release!](https://github.com/backdrop-contrib/webform_civicrm/releases/tag/1.x-5.0.0)

@herbdool, do I need to do anything else to trigger an update to https://backdropcms.org/project/webform_civicrm? If not, this issue can be closed!
